apps/webClient/src/views/Home/index.tsx

```diff
@@ -1,61 +1,139 @@
+import Box from "@material-ui/core/Box";
+import Button from "@material-ui/core/Button";
 import Typography from "@material-ui/core/Typography";
 import { Theme } from "@material-ui/core/styles/createMuiTheme";
-import Logo from "../../components/Logo";
 
 interface Props {}
 
-const backgroundImage = `${process.env.PUBLIC_BUCKET}/fd-home.jpg`;
+const logo = `${process.env.PUBLIC_BUCKET}/fd-logo.png`;
 
 const useStyles = makeStyles((theme: Theme) => ({
-  contentWrapper: {
-    paddingBottom: 0,
-    width: 176
-  },
-  top: {
-    backgroundImage: `url("${backgroundImage}")`,
-    height: "100%",
-    backgroundRepeat: "no-repeat",
-    backgroundPosition: "center",
-    backgroundColor: "black"
-  },
-  futuredays: {
+  logo: {
+    height: "100px",
+  },
+  logoContainer: {
+    display: "flex",
+    justifyContent: "center",
+  },
+  middleContainer: {
+    display: "flex",
+    justifyContent: "space-between",
+    marginTop: "80px",
+  },
+  lowerContainer: {
+    display: "flex",
+    justifyContent: "space-around",
+    marginTop: "80px",
+  },
+  heroContainer: {
+    maxWidth: "300px",
+  },
+  bodyText: {
+    color: theme.palette.secondary.main,
+    textAlign: "center",
+    fontStyle: "italic",
+  },
+  aboutUsButton: {
+    backgroundColor: "white",
+    borderRadius: 0,
+    width: "60px",
+    height: "200px",
+  },
+  ourWorkButton: {
+    backgroundColor: "transparent",
+    borderRadius: 0,
+    border: "solid 4px white",
+    width: "60px",
+    height: "200px",
+  },
+  aboutButtonText: {
+    writingMode: "vertical-rl",
+    textOrientation: "sideways",
+    fontWeight: 500,
+  },
+  ourWorkButtonText: {
+    // Need to swap facing direction
+    writingMode: "vertical-rl",
+    textOrientation: "sideways",
+    fontWeight: 500,
+  },
+  partnerCard: {
+    maxWidth: "300px",
+    textAlign: "center",
+  },
+  heroHeader: {
+    display: "flex",
+    justifyContent: "center",
+    alignItems: "flex-end",
     color: "white",
-    lineHeight: 1
-  },
-  titleContent: {
-    alignSelf: "flex-end",
-    marginBottom: theme.spacing(2),
-    marginRight: theme.spacing(2),
-    [theme.breakpoints.only("xs")]: {
-      backgroundColor: "black",
-      borderRadius: theme.spacing(2),
-      marginRight: 0
-    }
-  }
+    marginBottom: "10px",
+  },
+  whiteTitle: {
+    color: "white",
+    fontWeight: 500,
+    marginBottom: "10px",
+  },
 }));
 
 const Home: React.FC<Props> = ({}) => {
   const classes = useStyles();
-  const contentColor = "white";
 
   return (
-    <Grid container direction="row-reverse" className={classes.top}>
-      <Grid item className={classes.titleContent} sm={3}>
-        <Box pb={2} className={classes.contentWrapper}>
-          <Logo fill={contentColor} />
-          <Typography
-            className={classes.futuredays}
-            align="center"
-            variant="h5"
-          >
-            FutureDays
-          </Typography>
+    <Grid container style={{ margin: "8rem 0" }}>
```

Comment:
only use style when forced to.  I think we probably want to update this [file](https://github.com/cbaron/FutureDaysWebsite/blob/master/apps/webClient/src/components/View/index.tsx#L30)

as its a wrapper for all views.  To use a `Container` component and its `maxWidth` prop to give us the spacing we want.